projboard/tests/test_client.py

```diff
@@ -5,3 +5,9 @@
 def test_homepage(client):
     response = client.get("/")
     assert response.status_code == 200
```

Comment:
You are testing this change
path('', views.board, name='board') -> path('', views.board, name='board')

You are not suggest a test to verify the page itself
Please review UI talk, and search how to verify templates generated the html file

Reply:
@Yarboa You mean like to verify that the page contains the expected content ?
for example  `assert b"<h1>Smarticle</h1>" in response.content` ?

Reply:
No. search in the UI presentation the following:
```
 assert 'msgboard/post_list.html' in template_names
```